aws-rds-dbsubnetgroup/pom.xml

```diff
@@ -66,6 +66,12 @@
             <version>2.26.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
can you make changes in schema file to include "tagging" metadata information? wiki guide https://w.amazon.com/bin/view/AWS21/Design/Uluru/Onboarding_Guide/Tagging_Guide/, example CR: https://code.amazon.com/reviews/CR-59913948/revisions/1#/details